client/scss/styleguide/_styleguide.scss

```diff
@@ -270,3 +270,8 @@ html,
     display: block;
   }
 }
+
+.sample__component {
+  min-height: 100px;
+  background: olive;
+}
```

Comment:
I've had my editor add an empty newline to the end of files for ages because [this](http://stackoverflow.com/a/729795) – I guess you have the opposite.